docs/organizations/managing-people.md

```diff
@@ -12,11 +12,11 @@ For **private repositories**, Codacy only analyzes commits from people in your C
 To list and manage the people in your organization, open your organization **Settings**, page **People**. This page also shows their last activity on Codacy.
 
 !!! note
-    In some Enterprise plans, Codacy automatically manages people activity and seat usage for your organization:
+    In some Enterprise plans, Codacy automatically manages seat usage for your organization based on the contribution to your private repositories:
 
-    -   Adds new people who commit to your private repositories and analyzes their commits.
-    -   Deactivates people who perform no commit or login for 90 consecutive days.
-    -   Updates your organization seats accordingly when adding or deactivating people. Inactive people don't occupy a seat in your organization.
+    -   Adds new seats for people who commit to your private repositories and analyzes their commits.
```

Comment:
The sentence structure (the commit/analyzes combination) is a bit unclear. Could this make it clearer?

```suggestion
    -   Automatically adds a seat when analyzing the first contribution to your private repositories by a new person
```